src/resampling.jl

```diff
@@ -1,4 +1,5 @@
-export resample, resample_by_FFT, resample_by_RFFT, upsample2_abs2, upsample2, upsample2_1D
+export resample, resample_by_FFT, resample_by_RFFT, upsample2_abs2, upsample2, upsample2_1D, resample_var, resample_czt, barrel_pin
+using FastTransforms
```

Comment:
if you need it, put it in FourierTools.jl file

Reply:
I double checked. All other exports are also in the individual files.

Reply:
but not the using FastTransforms. That belongs in the FourierTools.jl file